code/_onclick/item_attack.dm

```diff
@@ -25,7 +25,13 @@
 	I.attack(src, user)
 
 /mob/living/proc/attacked_by(obj/item/I, mob/living/user, def_zone)
-	apply_damage(I.force, I.damtype, def_zone)
+	var/damage = I.force
+	if(mind && mind.demoninfo && mind.demoninfo.banetype == BANEWHITECLOTHES) // Sorry for the snowflake check.
+		if(istype(user, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = user
+			if(istype(H.w_uniform, /obj/item/clothing/under/color/white))
+				damage *= 3
+	apply_damage(damage, I.damtype, def_zone)

```

Comment:
Yeah, it's making white clothed people do more damage to demons with the appropriate bane.  I'm still wracking my brain for a better way to do this.  I'm thinking add a whiteness value to clothing, and then add a check_weakness method to mob/living?